### DISCUSSION TITLE
Welcome to eegUtils Discussions!

### DISCUSSION THREAD

**Discussion by craddm** [craddm/eegUtils#115] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by rgutierrezsampson** [comment]
Question using Topoplot: when importing the data (specifically from excel), how should the data be formatted so that Topoplot can read the input?

e.g., how I'm importing currently

Channel | Value
F3 | -10.211879
F4 | -23.680149
P7 | 9.14755405
P8 | -21.595894
T7 | 1.17814788
T8 | 3.16632504
O1 | -6.2023184
O2 | 2.63248783

can someone give me an example of how the excel data should be organized in order for eegutil to read it?
